src/View/RssView.php

```diff
@@ -310,7 +310,9 @@ protected function _prepareOutput($item) {
 						}
 						$val = $attrib;
 					} elseif (is_array($val) && isset($val['url'])) {
-						$val['url'] = Router::url($val['url'], true);
+						if (!isset($val['@isPermalink']) || $val['@isPermalink'] === "true") {
```

Comment:
I wonder if we should use `$val['@isPermalink'] !== "false"` here instead as the current default is true anyway (if you dont include a `@isPermalink` key.